lib/install/management/dispatcher.go

```diff
@@ -125,12 +139,108 @@ func (d *Dispatcher) initDiagnosticLogs(conf *metadata.VirtualContainerHostConfi
 	}
 }
 
+func (d *Dispatcher) registerExtension(conf *metadata.VirtualContainerHostConfigSpec) error {
+	log.Infoln("Registering VCH as a vSphere extension")
+	extensionManager := object.NewExtensionManager(d.session.Vim25())
+	extension := types.Extension{
+		Description: &types.Description{
+			Label:   "VIC",
+			Summary: "vSphere Integrated Containers Virtual Container Host",
+		},
+		Company: "VMware, Inc.",
+		Version: "0.2",
+		Key:     "com.vmware.vic",
+	}
+
+	// anonymous function generates a keypair
+	var err error
+	if conf.ExtensionCert, conf.ExtensionKey, err = func() (string, string, error) {
```

Comment:
Don't see a need to use an anonymous func here.. can this func be moved elsewhere to its own named function or method?